corehq/apps/reports/standard/cases/basic.py

```diff
@@ -46,8 +46,11 @@ class CaseListMixin(ESQueryProfilerMixin, ElasticProjectInspectionReport, Projec
     search_class = case_es.CaseES
 
     def _base_query(self):
+        search_class = self.search_class()
+        if self.rendered_as == 'export':
+            search_class = self.search_class(for_export=True)
```

Comment:
Not that it makes a huge difference, but I think it would be clearer (and FWIW execute one fewer line) if you had this be an if else
```suggestion
        if self.rendered_as == 'export':
            search_class = self.search_class(for_export=True)
        else:
            search_class = self.search_class()
```